lib/Conversation/styles.scss

```diff
@@ -57,7 +64,7 @@ $conversation-meta-text-color: $color-brand-dark !default;
 }
 
 .conversation__reply-count {
-  padding: 0 0 $conversation-padding $conversation-padding+$avatar-size;
+  padding: 0 0 $conversation-padding $conversation-padding+$avatar-size+10;
```

Comment:
Look at the "setting" feedback about regarding the hardcoded spacing extension of `10`.